app/views/stafftools/assignment_invitations/_assignment_invitation.html.erb

```diff
@@ -7,9 +7,9 @@
 
   <div class="col-10 col-md-10 d-flex flex-column">
     <%= link_to stafftools_assignment_invitation_path(assignment_invitation.id) do %>
-      <strong class="search-result-title">Assignment invitation for <%= assignment_invitation.assignment_title %></strong>
+      <strong class="search-result-title"><%= t('views.stafftools.assignment_invitations.invitation_for') + assignment_invitation.assignment_title %></strong>
```

Comment:
Let's interpolate this instead of merging it together

```erb
<strong class="search-result-title"><%= "#{t('views.stafftools.assignment_invitations.invitation_for')} #{assignment_invitation.assignment_title}" %></strong>
```

And remove the trailing space here: https://github.com/education/classroom/pull/941/files#diff-d0df02bbf4aa5432e92d78e907528becR5